NetFlow/NetFlow.manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "author": "NetFlow Logic",
+    "categories": [
+        "Compliance and Security",
+        "IT Infrastructure",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "The NetFlow Optimizer app provides dashboards to monitor your network traffic in data centers or clouds providing details on inbound/outbound accepted rejected flows, source and destination IP addresses, ports, protocols, as well as usage by application and user. Data is imported into Sumo via JSON streaming of NetFlow Optimizer processing NetFlow, IPFIX, sFlow, and cloud VPC flow logs.",
+    "helpURL": "https://docs2.netflowlogic.com/integrations-and-apps/integration-with-sumo-logic/",
```

Comment:
Nice job. I just think correct link is:
https://docs.netflowlogic.com/integrations-and-apps/integration-with-sumo-logic/

Reply:
confirmed, it was not updated